catalog/spatial/registry/registry-policy-plugin/src/main/java/org/codice/ddf/registry/policy/RegistryPolicyPlugin.java

```diff
@@ -159,7 +159,7 @@ private void parsePermissionsFromString(List<String> permStrings,
 
     private PolicyResponse getWritePolicy(Metacard input, Map<String, Serializable> properties) {
         HashMap<String, Set<String>> operationPolicy = new HashMap<>();
-        if (Requests.isLocal(properties) && RegistryUtility.isRegistryMetacard(input)) {
+        if (Requests.isLocal(properties) && input != null && RegistryUtility.isRegistryMetacard(input)) {
```

Comment:
Yeah @coyotesqrl good point. I am little confused as to why a null metacard would be getting passed in here. Handling null is good, but I'm wondering if we need to look at the bigger flow.